modules/configuration/ajax/updateProject.php

```diff
@@ -52,6 +54,18 @@
         ),
         array("ProjectID" => $_POST['ProjectID'])
     );
+    $projectID = $_POST['ProjectID'];
+}
+
+$db->delete('project_subproject_rel', array('ProjectID' => $projectID));
+foreach ($_POST['SubprojectIDs'] as $sid) {
+    $db->insertIgnore(
+        'project_subproject_rel',
+        array(
+         'ProjectID'    => $projectID,
+         'SubprojectID' => $sid,
+        )
+    );
```

Comment:
This will have to be modified to only delete those not needed anymore and to add only the new one.

Reply:
@PapillonMcGill this change was done in the PR linked above. I will just need to rebase this one 👍

Reply:
Ok, I will then approve this PR after the rebase